ToDo-Garden/ToDo-Garden/SettingScene/Sources/SettingScene/SettingRouter.swift

```diff
@@ -69,12 +77,9 @@ extension SettingRouter: SettingRoutingLogic {
     guideSceneViewController.modalPresentationStyle = .fullScreen
     self.viewController?.navigationController?.pushViewController(guideSceneViewController, animated: true)
   }
-}
-
-// MARK: - Declare Payload for scene
-
-extension SettingRouter {
-	struct NextScenePayload: NextScenePayloadable {
-		// var name: String
-	}
+  
+  func routeToUserInfoScene() {
+    let userInfoScene = self.userInfoSceneBuilder.build()
+    self.viewController?.navigationController?.pushViewController(userInfoScene, animated: true)
+  }
```

Comment:
```suggestion
  
  func routeToUserInfoScene() {
 self.viewController?.navigationController?.pushViewController(self. userInfoSceneViewController, animated: true)
  }
```

route마다 매번 build()를 호출하여 생성하게 되니까 메모리 관리가 잘 안되더라고요. 이 방식으로 가는게 좋을 것 같습니다. 
저도 이미 해놓은것들도 이 방식으로 바꿀예정.

Reply:
아래에 반영했습니다!
https://github.com/ToDo-Garden/ToDo-Garden/pull/902/commits/bc5a59c4f351f5d95ba9a1e4ef936946e87ba56d